src/uct/cuda/gdr_copy/Makefile.am

```diff
@@ -24,6 +24,9 @@ libuct_cuda_gdrcopy_la_SOURCES = \
 	gdr_copy_iface.c \
 	gdr_copy_ep.c
 
+PKG_CONFIG_NAME=cuda-gdrcopy
```

Comment:
it should be "gdrcopy" ; not "cuda-gdrcopy"

Reply:
> it should be "gdrcopy" ; not "cuda-gdrcopy"

renamed